python/cuml/tsa/arima.pyx

```diff
@@ -196,9 +234,20 @@ class ARIMAModel(Base):
                  handle=None):
         super().__init__(handle)
         self.order = order
-        self.mu = mu
-        self.ar_params = ar_params
-        self.ma_params = ma_params
+
+        # Convert the parameters to numpy arrays if needed
+        if type(mu) is list:
+            self.mu = np.array(mu)
+        else:
+            self.mu = mu
+        if type(ar_params) is list:
+            self.ar_params = np.transpose(ar_params)
+        else:
+            self.ar_params = ar_params
+        if type(ma_params) is list:
+            self.ma_params = np.transpose(ma_params)
+        else:
+            self.ma_params = ma_params
```

Comment:
Couldn't this use `input_to_host_array` to also convert order if necessary for numpy arrays, and in the case the user (for whatever reason) already has data in device?

Reply:
I didn't even know that this function existed, thanks. We still need to check if the input is a list and handle the list differently, right?

I'll make this change for now, and ideally these arrays will become device arrays as soon as lbfgs-b moves to the CUDA side.

Reply:
Fixed